velox/vector/DecodedVector.h

```diff
@@ -84,6 +132,11 @@ class DecodedVector {
 
   /// Given a dictionary vector with at least 'numLevel' levels of dictionary
   /// wrapping, combines 'numLevel' wrappings into one.
+  /// Example usage: <InputVector> : Dictionary2(Dictionary1(Complex)) If we
+  /// call makeIndices(<Input Vector>, Level = 2), only Dictionary2 and
+  /// Dictionary1 will have indices and nulls merged and the nulls from the
+  /// Complex flat vector will not be merged. If instead Level is set to 3, then
+  /// additionally nulls from the complex base vector will also be merged.
```

Comment:
Thanks for the comment! I didn't know the difference between numLevels being 2 and 3. (I knew the difference between 1 and 2 though.)